dask_kubernetes/operator/__init__.py

```diff
@@ -1,3 +1,5 @@
+from .controller import *
```

Comment:
@jacobtomlinson I think this might be related to https://github.com/dask/dask-kubernetes/pull/649#issuecomment-1439625841 as otherwise tests won't run on M1 for some strange reason

Reply:
Fair enough. Would you mind opening a separate issue to discuss M1 development though?